From 74e72e20bac15d6004822bb5aedefd7aa7ff5417 Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan.agner@toradex.com>
Date: Thu, 11 Jul 2019 14:01:48 +0200
Subject: [PATCH] apalis-imx8: load HDMI firmware and use distro boot by
 default

Load HDMI firmware by default before running distro boot.

Signed-off-by: Stefan Agner <stefan.agner@toradex.com>
---
 include/configs/apalis-imx8.h | 16 ++--------------
 1 file changed, 2 insertions(+), 14 deletions(-)

diff --git a/include/configs/apalis-imx8.h b/include/configs/apalis-imx8.h
index 79d26fca53..98b34d2574 100644
--- a/include/configs/apalis-imx8.h
+++ b/include/configs/apalis-imx8.h
@@ -111,6 +111,7 @@
 	"fdt_addr=0x83000000\0"			\
 	"boot_fdt=try\0" \
 	"fdt_file=fsl-imx8qm-apalis.dtb\0" \
+	"fdtfile=fsl-imx8qm-apalis.dtb\0" \
 	"initrd_addr=0x83800000\0"		\
 	"initrd_high=0xffffffffffffffff\0" \
 	"mmcdev="__stringify(CONFIG_SYS_MMC_ENV_DEV)"\0" \
@@ -162,21 +163,8 @@
 		"fi;\0"
 
 #undef CONFIG_BOOTCOMMAND
-#ifdef CONFIG_TDX_EASY_INSTALLER
-#define CONFIG_BOOTCOMMAND "run distro_bootcmd"
-#else
 #define CONFIG_BOOTCOMMAND \
-	   "mmc dev ${mmcdev}; if mmc rescan; then " \
-		   "if run loadbootscript; then " \
-			   "run bootscript; " \
-		   "else " \
-			   "if run loadimage; then " \
-				   "run mmcboot; " \
-			   "else run netboot; " \
-			   "fi; " \
-		   "fi; " \
-	   "else booti ${loadaddr} - ${fdt_addr}; fi"
-#endif
+	   "run loadhdp; hdp load ${hdp_addr}; run distro_bootcmd"
 
 /* Link Definitions */
 #define CONFIG_LOADADDR			0x80280000
-- 
2.22.0

